driver/ppm_events.c

```diff
@@ -349,7 +349,17 @@ inline u32 compute_snaplen(struct event_filler_arguments *args, char *buf, u32 l
 						dport = 0;
 					}
 
-					if (sport == PPM_PORT_MYSQL || dport == PPM_PORT_MYSQL) {
+					if (args->consumer->fullcapture_port_range_end != 0 &&
```

Comment:
Introduce some variables for `args->consumer->fullcapture_port_range_end` and `args->consumer->fullcapture_port_range_start`? This is pretty unreadable.

Or maybe:

```
static inline bool in_port_range(uint16_t port, uint16_t min, uint16_t max)
{
    return port <= min && port >= max;
}

...

uint16_t min_port = args->consumer->fullcapture_port_range_start;
uint16_t max_port = args->consumer->fullcapture_port_range_end;

if (in_port_range(sport, min_port, max_port) ||
    in_port_range(dport, min_port, max_port)
{
    // ...
}
```